templates/activity/show.html.twig

```diff
@@ -20,10 +20,11 @@
             <span>{{ activity.start|format_datetime('medium', 'short') }} - {{ activity.end|format_datetime('medium', 'short') }}</span>
         {% endif %}
         <span>{{ activity.location.address }}</span>
+        {% if activity.author %}
```

Comment:
I can't see a situation where it is desirable to have an activity without an author.
Curious what your thought process was.

Reply:
One can use Kiwi without utilizing the group system (eg. only admins), this check reflects that possibility. That's something we could change, but I feel that would be outside of the scope of this PR.

Reply:
hadn't even considered that, my bad.
I do agree that such changes would be outside the scope of this 3 line PR.

I'll merge